gclient/src/utils.rs

```diff
@@ -19,27 +19,47 @@
 use crate::{Error, Result};
 use std::{
     fs,
-    path::PathBuf,
+    path::Path,
     time::{SystemTime, UNIX_EPOCH},
 };
 use wabt::Wat2Wasm;
 
+/// Compile a source WebAssembly Text (WAT) to Wasm binary code.
 pub fn wat2wasm(wat: impl AsRef<str>) -> Vec<u8> {
     Wat2Wasm::new()
         .convert(wat.as_ref())
-        .expect("Failed to parse WASM")
+        .expect("Failed to parse WAT")
         .as_ref()
         .to_vec()
 }
 
-pub fn bytes_now() -> [u8; 16] {
+/// Return the time elapsed since the Unix epoch in microseconds.
+pub fn now_in_micros() -> u128 {
     SystemTime::now()
         .duration_since(UNIX_EPOCH)
         .expect("Infallible")
         .as_micros()
-        .to_le_bytes()
 }
 
+/// Return the time elapsed since the Unix epoch in microseconds converted to
+/// the 16-bytes array.
+#[deprecated = "Use `now_in_micros().to_le_bytes()` instead"]
```

Comment:
ping ecosystem that this method will be removed soon

Reply:
Yes, of course